core-blocks/pullquote/index.js

```diff
@@ -60,25 +54,15 @@ export const settings = {
 
 	attributes: blockAttributes,
 
-	getEditWrapperProps( attributes ) {
-		const { align } = attributes;
-		if ( 'left' === align || 'right' === align || 'wide' === align || 'full' === align ) {
-			return { 'data-align': align };
-		}
+	supports: {
+		align: true,
 	},
 
 	edit( { attributes, setAttributes, isSelected, className } ) {
-		const { value, citation, align } = attributes;
-		const updateAlignment = ( nextAlign ) => setAttributes( { align: nextAlign } );
+		const { value, citation } = attributes;
 
 		return (
 			<Fragment>
```

Comment:
`Fragment` wrapper is no longer necessary in here.

Reply:
Addressed in [this](https://github.com/WordPress/gutenberg/pull/7100/commits/9374ed318ead245d1b3ba974307c1c7a62bc7096) commit. Thank you ❤️